src/helpers/Available.js

```diff
@@ -0,0 +1,10 @@
+import Amountsold from "./Amountsold.js";
+import Amountbought from "./Amountbought.js";
+
+
+function Available() {
+
+   return Amountbought() - Amountsold()
```

Comment:
Ik zou dit wat explicieter maken
```suggestion
   return Amountbought() - Amountsold() > 0;
```